### PR TITLE
sol andrii

### PR DESCRIPTION
**Pull request by LozytskyiA**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  removeNumber(n) {
+    let index = this.num.indexOf(n);
+    if(index === -1){
+      return;
+    }
+    this.num.splice(index, 1);
+  }
+  sum(){
+    if(this.num.length !== 0) {
+      return this.num.reduce(((a, b) => a + b),0);
```
Comment (gestap) [inline main.js]:
```suggestion
      return this.num.reduce(((a, b) => a + b), 0);
```

File: main.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+  }
+  product() {
+    return this.num.reduce(((a, b) => a * b),1);
```
Comment (gestap) [inline main.js]:
```suggestion
    return this.num.reduce(((a, b) => a * b), 1);
```

File: main.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  }
+  min(){
+    return Math.min(...this.num);
+  }
+  max(){
+    return Math.max(...this.num);
+  }
+}
```
Comment (gestap) [inline main.js]: the document must end with one empty line

File: main.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  }
+  removeNumber(n) {
+    let index = this.num.indexOf(n);
+    if(index === -1){
+      return;
+    }
+    this.num.splice(index, 1);
+  }
+  sum(){
+    if(this.num.length !== 0) {
```
Comment (gestap) [inline main.js]: no need for this checking

**Review by ost-ptk — CHANGES_REQUESTED** [review]

File: main.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    this.num.splice(index, 1);
+  }
+  sum(){
+    return this.num.reduce(((a, b) => a + b), 0);
+  }
+  product() {
+    return this.num.reduce(((a, b) => a * b), 1);
+  }
+  min(){
+    return Math.min(...this.num);
```
Comment (gestap) [inline main.js]: `...or throws an error if there are no numbers.`

File: main.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  }
+  max(){
+    return Math.max(...this.num);
```
Comment (gestap) [inline main.js]: `...or throws an error if there are no numbers.`

**Review by ost-ptk — CHANGES_REQUESTED** [review]

**Review by ost-ptk — APPROVED** [review]

**Pull request closed without merging**